setup-hakn/action.yaml

```diff
@@ -0,0 +1,178 @@
+# Copyright 2022 Chainguard, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+name: 'Setup chainguard-dev/hakn'
+description: |
+  This action sets up chainguard-dev/hakn Knative on the current
+  kubectl context (typically KinD from setup-kind).
+
+inputs:
```

Comment:
we should make the version of istioctl version configurable

Reply:
Done